lib/utils/scroll_behavior.dart

```diff
@@ -28,7 +28,7 @@ class BouncingScrollWrapper extends StatelessWidget {
   Widget build(BuildContext context) {
     return ScrollConfiguration(
       behavior: BouncingScrollBehavior(),
-      child: child,
+      child: child!,
```

Comment:
As `ScrollConfiguration` requires a child, maybe `BouncingScrollWrapper` should as well? Or at least check for nulabillity before using the bang operator, returning a `SizedBox()` otherwise.